test/Interop/Cxx/members/emit-function-from-called-method-irgen.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+
+import MethodCallsFunction
+
+public func getValueFromMethod() -> CInt {
```

Comment:
Maybe check the body of `getValueFromMethod `/`callIncrement `/`increment`? Right now it's not really clear to me that this is doing the correct thing (other than the fact that `increment` is *define*d).

Reply:
What we are currently running into is a 'declare' statement for the inline function as opposed to a definition. I believe checking for the 'define' line is sufficitent. Checking that the correct IR is generated for inline function decls is out of scope of this PR and hopefully tested somewhere else :)

Reply:
Given that this issue existed, unfortunately, I'm not sure it is tested elsewhere. I think it would be good to check that `getValueFromMethod` produces the correct IR (for example, we need to make sure that `callMethod` was inlined, and the `this` argument was passed correctly). I'm happy to submit a follow-up PR or commit to this PR with the tests if you'd rather. But I think they should exist.